code/game/mecha/mecha.dm

```diff
@@ -664,6 +664,10 @@
 		user << "<span class='warning'>Access denied.</span>"
 		log_append_to_last("Permission denied.")
 		return
+	if(user)(mob_buckled)

```

Comment:
rip, Would this create an error, even if it works for both the player and if someone tried to do this with a mob?